src/components/LangDropdown.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import "./langDropdown.css";
+
+function LangDropdown() {
+  const { t, i18n } = useTranslation();
```

Comment:
doesn't @gbilgedal have a separate lang selector? what's this file

Reply:
@louisrli when I solved the conflicts, this file was one of the conflict files, I just accept incoming changes, because I was merging the old dev version
Also as a result all the conflict files was added as (Files changed) in this PR, although I didn't do anything to them on my local repo, just fixed the conflicts here on the remote repo

Reply:
I think that probably it's being duplicated, you should check where the real LangDropdown is and work with @gbilgedal please. Basically, in the end the PR should be clean without this file :) the changes should be isolated only to the actual difference

Reply:
@louisrli PTAL 
I fixed the issue after merging the last PR
Now they are only 5 changed files